### DISCUSSION TITLE
How to pin or select version of lsp server to install?

### DISCUSSION THREAD

**Discussion by zer09** [mason-org/mason.nvim#1745] [Q&A]
Instead of installing the latest version of lsp from the `mason-registry`, how I can pin lsp version or rollback the old version?

**Comment by spvky** [comment]
This is pretty old but in case anyone runs into this you can run `MasonInstall <lsp-client>@<version tag>` to install a specific version of a language server, but I'm not sure how to pin a version

**Reply by rodhash** [reply]
Thank you

It'd be nice if this syntax worked in the `ensure_installed` as well but apparently it doesnt.

**Comment by rodhash** [comment]
> Instead of installing the latest version of lsp from the `mason-registry`, how I can pin lsp version or rollback the old version?

any luck finding a way to pin a specific version? Today the newest version of js-debug adapter seems to be broken so I had to rollback one version.
